Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent any
+    environment {
+        ECR="018895040333.dkr.ecr.us-east-1.amazonaws.com"
+        IMAGE="imply-eng/jfrog-mvn-java8:20201009-033920"
+    }
+    stages {
+
+        stage('Pull Image') {
+            steps {
+                withDockerRegistry([url: "https://$ECR", credentialsId: "ecr:us-east-1:aws"]) {
+                    sh "docker pull $ECR/$IMAGE"
+                }
+            }
+        }
+
+        stage ('Build and Publish') {
+            steps {
+                sh """
+                    echo "mvn versions:set -B -DremoveSnapshot | true" > run.sh
+                    echo "jfrog rt mvn clean install" >> run.sh
+                    docker run -v ${WORKSPACE}/.m2:/root/.m2 -v ${WORKSPACE}:/app $ECR/$IMAGE bash run.sh
+                """
+            }
```

Comment:
How is the snapshot version bumped after the artifact is built and published?

Reply:
I thought you don't need snapshot versions.
But here is what I tested.
https://artifactory.qa.imply.io/artifactory/webapp/#/artifacts/browse/tree/General/imply-libs-snapshot-local/io/netty/netty/3.10.6.Final-iap3-SNAPSHOT
![image](https://user-images.githubusercontent.com/70178960/95621833-37bbf800-0a27-11eb-80bf-7d402b2f5509.png)

Reply:
We don't need SNAPSHOTS to be published, but can you describe the high level workflow of adding fixes to the netty repo, and how they get published with a new version and then how Druid will pick up the new version

Reply:
I am updating the build process to use our new artifactory at https://implydata.jfrog.io/, will add readme asap.

Reply:
The readme is updated